### ISSUE TITLE
Bug: Free XP with Actually Additions Solidified Experience

### ISSUE THREAD

**Issue by Rated01010010** [Ipsis/Woot#83]
Actually Additions has a feature where mobs randomly drop solidified experience when killed. Well in Woot, it just spits these out in the air infinitely without any power going to it. 

![scrot033023](https://cloud.githubusercontent.com/assets/20819163/17455481/0f958286-5b87-11e6-8ec4-46e3147df831.jpg)

**Comment by Ipsis** [comment]
It is a problem with a few "naughty" mods that generate their loot in the world, rather than adding it to the mob death loot list.
The next version I'm releasing, will actually hoover these items up during the learning phase.
See https://github.com/Ipsis/Woot/issues/75

**Comment by Ellpeck** [comment]
So what, do I need to do anything or is this fine now?

**Comment by Ipsis** [comment]
Ellpeck - apologies, I had no idea that this was mentioned against your mod as well.

Some quick background - since I used the word naughty.
On the LivingDropsEvent there are two options.
* add the items you want the mob to drop to the event drop list and then MC will spawn all the items in the drop list in the world for you
* manually spawn your items in the world

Both ways are acceptable, I just happen to kill mobs that don't exist in the world to learn what kind of loot they drop, therefore when I'm doing that mods that use the second option spawn their loot in the world.

I've added code to try and cope with mods using the second option, since there are quite a few of them.

So there is nothing for you to do, it is all down to me to handle the problem that I create :)
